File: ex02/ex02_06/ex02_06.py
citizen =  {
  "name": "",
  "age": 0,
  "sex": "",
  "id": "",
}

def main():
  citizen['name'] = input("Enter name: ")
  citizen["age"] = input("Enter age: ")
  citizen["sex"] = input("Enter sex (F/M): ")
  citizen["id"] = input("Enter ID: ")
  print("*********************")
  print("Name: " + citizen["name"])
  print("Age: " + str(citizen["age"]))
  print("Sex: " + citizen["sex"].upper())
  print("ID: " + citizen["id"])
  return 0

if __name__ == "__main__":
  main()

File: ex02/ex02_06/ex02_06.js
import promptSync from 'prompt-sync';
const prompt = promptSync();

let citizen =  {
  name: "",
  age: 0,
  sex: undefined,
  id: "",
}

function main() {
  citizen.name = prompt("Enter name: ");
  citizen.age = prompt("Enter age: ");
  citizen.sex = prompt("Enter sex (F/M): ");
  citizen.id = prompt("Enter ID: ");
  console.log("*********************");
  console.log("Name: " + citizen.name);
  console.log("Age: " + String(citizen.age));;
  console.log("Sex: " + citizen.sex.toUpperCase());
  console.log("ID: " + citizen.id);
  return 0;
}
main();